conductor_api/src/interface.rs

```diff
@@ -715,6 +715,61 @@ impl ConductorApiBuilder {
         self
     }
 
+    /// Adds functions useful for debugging.
+    ///
+    /// - `debug/running_instances`
+    ///     Install a UI bundle that can later be hosted by an interface
+    ///     Params:
+    ///     - `id` ID used to refer to this bundle
```

Comment:
this looks like comment mistake? left over from a copy?  I think there are no params.